stores/metadata.go

```diff
@@ -818,7 +914,7 @@ func (s *SQLStore) SearchObjects(ctx context.Context, substring string, offset,
 		Table("objects o").
 		Joins("LEFT JOIN slices sli ON o.id = sli.`db_object_id`").
 		Joins("LEFT JOIN slabs sla ON sli.db_slab_id = sla.`id`").
-		Where("INSTR(o.object_id, ?) > 0", substring).
+		Where("INSTR(o.object_id, ?) > 0 AND ?", substring, sqlWhereBucket("o", bucket)).
```

Comment:
the `sqlWhereBucket` feels a bit odd? why not just `Join` and do it explicitly? I see why that could be useful but adding this here just makes the store more inconsistent and it's a bit sneaky to hide a join like that I think